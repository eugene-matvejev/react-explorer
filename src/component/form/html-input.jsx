import React from 'react';
import composeInput from './compose-form-field';

const HTMLInput = (props) => <input {...props} />;

HTMLInput.propTypes = {
};
HTMLInput.defaultProps = {
};

export default composeInput(HTMLInput);
export {
    HTMLInput as PureHTMLInput
}
