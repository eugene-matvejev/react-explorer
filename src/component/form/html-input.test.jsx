import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { PureHTMLInput as HTMLInput } from './html-input';

configure({ adapter: new Adapter() });

describe('<HTMLInput/>', () => {
    const props = {
    };

    describe('render', () => {
        it('with default/required props', () => {
            const c = shallow(<HTMLInput {...props} />);

            expect(c).toMatchSnapshot();
        });
    });
});
