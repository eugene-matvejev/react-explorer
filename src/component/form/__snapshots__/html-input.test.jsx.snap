// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HTMLInput/> render with default/required props 1`] = `<input />`;
